distribution/src/release/LICENSE

```diff
@@ -201,9 +201,6 @@
    limitations under the License.
 
 
-===========================================================================
-For alpha-server-${version}-exec.jar
```

Comment:
As we have the distribution of the alpha-server, we still need to keep this part in the license.